README.md

```diff
@@ -136,51 +144,73 @@ Scanner vendors are supposed to implement Scanner Adapters exposing the Scanner
          "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
        }
        ```
-3. Try getting scan report (in unified Harbor format understandable by Harbor Web console):
+3. Try getting scan report (in the new v1.1 format understandable by Harbor Web console):
    ```
-   curl -H 'Accept: application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0' \
+   curl -H 'Accept: application/vnd.security.vulnerability.report; version=1.1' \
      http://scanner-adapter:8080/api/v1/scan/3fa85f64-5717-4562-b3fc-2c963f66afa6/report
 
    Retry-After: 15
    Status: 302 Found
    ```
 4. Wait 15 seconds or use your own retry interval ...
 5. ... and try getting scan report again:
+   
    ```
-   curl -H 'Accept: application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0' \
-     http://scanner-adapter:8080/api/v1/scan/3fa85f64-5717-4562-b3fc-2c963f66afa6/report
-
-   Content-Type: application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0
-   Status: 200 OK
-
-   {
-     "generated_at": "2019-08-07T12:17:21.854Z",
-     "artifact": {
-       "repository": "library/mongo",
-       "digest": "sha256:917f5b7f4bef1b35ee90f03033f33a81002511c1e0767fd44276d4bd9cd2fa8e"
-     },
-     "scanner": {
-       "name": "Microscanner",
-       "vendor": "Aqua Security",
-       "version": "3.0.5",
-     },
-     "severity": "High",
```

Comment:
Why are we deleting those properties? I think it's useful to keep them as reference